### PR TITLE
Lvl 4, task 2, give snow in reward

### PR DESCRIPTION
**Pull request by Luciogi**

### CODE REVIEW

**Review by NoNameDude — COMMENTED** [review]
Why exactly snow?

File: skyblock/skyblock_levels/skyblock.levels.4.lua [hunk 1]
```diff
@@ -13,7 +13,7 @@ Level 4 mostly revolving around farming and dying
 level 4 feats and rewards:
 
 * craft_diamondhoe       farming:seed_wheat
-* use_hoe x40            farming:melon_slice
+* use_hoe x40            default:snow x9
 * plant_wheatseed        default:cactus
 * craft_flour            farming:seed_cotton x10
 * place_snowblock x50    dye:red x20
```
Comment (Luciogi) [inline skyblock/skyblock_levels/skyblock.levels.4.lua:19]: this task need snow

Comment (Luciogi) [reply skyblock/skyblock_levels/skyblock.levels.4.lua:19]: https://github.com/SkytopiaMinetest/worldmods-Skytopia/pull/1/files#diff-1a994dd487430b574ec44a76a0fcd7b305920d59bab278fc90a93761571738cbL19

Comment (NoNameDude) [reply skyblock/skyblock_levels/skyblock.levels.4.lua:19]: Understand

**Pull request merged**